tests/testapp/tests.py

```diff
@@ -339,17 +336,6 @@ def test_login_page_is_set_up_as_expected(self):
         self.assertTrue(
             isinstance(form, AuthenticationForm), type(form).__mro__)
 
-    @patch('django.contrib.auth.views.login')
-    def test_act_as_login_view_protects_sensitive_vars(self, mocked_login):
```

Comment:
thinking out loud: shouldn't this test be kept (adjusted) for regression? or do we implicitly trust Django upstream there won't be a regression? a quick check of https://github.com/django/django/blob/master/tests/auth_tests/test_views.py doesn't show any tests there (could be elsewhere). Maybe contribute it? The code does have `dispatch` wrapped in it https://github.com/django/django/blob/master/django/contrib/auth/views.py#L49

not sure how I would test it though - add a logger in test and capture log/stdout and assert password is not there, but username is? or can we introspect the wrapper to say it is the same as the sensitive_variables wrapper?

Reply:
Testing something you don't own is brittle. Testing some unchanged functionality in a dependent package feels wrong.

As you suggested, this should be a unit test on Django Auth. You could also place it in an integration test as a regression.